content/en/docs/ops/configuration/traffic-management/dns-proxy/index.md

```diff
@@ -135,10 +141,76 @@ $ kubectl exec deploy/sleep -- curl -sS -v auto.internal
 
 As you can see, the request is sent to an automatically allocated address, `240.240.0.1`. These addresses will be picked from the `240.240.0.0/16` reserved IP address range to avoid conflicting with real services.
 
+## External TCP services without VIPs
+
+Istio by default has a limitation on routing external TCP traffic, because it is not able to distinguish between two different TCP services. This limitation especially impacts the use of third party databases, such as AWS Relational Database Service or any database setup with geographical redundancy. Similar, but different external TCP services, cannot be handled separately by default. For the sidecar to accurately distinguish traffic between two different TCP services that are outside the mesh, the services must be on different ports or they need to have a globally unique VIP. For example, if you have two external database services, `mysql-instance1` and `mysql-instance2`, and you create service entries for them, the sidecar will still have a single listener on `0.0.0.0:{port}` that looks up the IP address of only `mysql-instance1` from public DNS servers and forwards traffic to it. It cannot route traffic to `mysql-instance2` as it has no way of distinguishing whether traffic arriving at `0.0.0.0:{port}` is bound for `mysql-instance1` or `mysql-instance2`.
+
+The below example shows how we can make use of DNS proxying as a solution to this problem. A virtual IP address is assigned to the service entries automatically in this case, so that sidecar can clearly distinguish traffic bound for each external TCP service.
```

Comment:
```suggestion
The following example shows how DNS proxying can be used to solve this problem.
A virtual IP address will be assigned to the every service entry so that client sidecars can clearly distinguish traffic bound for each external TCP service.
```